utils/oscap-podman

```diff
@@ -99,9 +99,8 @@ if [ ! -f "$DIR/run/.containerenv" ]; then
     touch "$DIR/run/.containerenv"
 fi
 
-for VAR in `podman inspect $ID --format '{{join .Config.Env " "}}'`; do
-    eval "export OSCAP_OFFLINE_$VAR"
-done
+export OSCAP_CONTAINER_VARS
+OSCAP_CONTAINER_VARS=`podman inspect $ID --format '{{join .Config.Env "\n"}}'`
```

Comment:
What is the reason of this change? I am not against, just asking.

Reply:
The idea is to create an independent code branch for consuming sideline source of environment variables instead of polluting process variables when we are in offline mode but don't have a running system. This is kind of special things among probes and hopefully it will evolve with introduction of system meta information.

Reply:
See https://github.com/OpenSCAP/openscap/pull/1499/files#diff-c414bc5e3aa458311c624f11370a735fR107

Reply:
Thanks, makes sense.